.travis.yml

```diff
@@ -5,6 +5,11 @@ python:
   - "3.6"
   - "3.7"
 
+notifications:
+  webhooks: https://www.travisbuddy.com/
+  on_success:never
+
+
```

Comment:
let's avoid this extra blank line please?

Reply:
Done